maxim/logger/mistral/utils.py

```
@@ -0,0 +1,151 @@
+import time
+from typing import Any, Dict, Iterable, List, Optional
+
+from mistralai.models import ChatCompletionResponse, CompletionEvent, CompletionChunk
+
+from ..logger import GenerationRequestMessage
+
+
+class MistralUtils:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)